tools/linter/adapters/ufmt_linter.py

```diff
@@ -2,20 +2,60 @@
 
 import argparse
 import concurrent.futures
+import fnmatch
 import json
 import logging
 import os
+import re
 import sys
 from enum import Enum
 from pathlib import Path
 from typing import Any, NamedTuple
 
+import isort
+from isort import Config as IsortConfig
 from ufmt.core import ufmt_string
 from ufmt.util import make_black_config
 from usort import Config as UsortConfig
 
 
 IS_WINDOWS: bool = os.name == "nt"
+REPO_ROOT = Path(__file__).absolute().parents[3]
+ISORT_WHITELIST = re.compile(
+    "|".join(
+        map(
+            fnmatch.translate,
+            [
+                "**",
+                ".ci/**",
+                ".github/**",
+                "benchmarks/**",
+                "functorch/**",
+                "tools/**",
+                "torchgen/**",
+                "test/**",
+                "test/[a-b]*/**",
+                "test/d*/**",
+                "test/dy*/**",
+                "test/[e-h]*/**",
+                "test/i*/**",
+                "test/j*/**",
+                "test/[k-p]*/**",
+                "test/[q-z]*/**",
+                "torch/**",
+                "torch/_[a-c]*/**",
+                "torch/_d*/**",
+                "torch/_[e-h]*/**",
+                "torch/_i*/**",
+                "torch/_[j-z]*/**",
+                "torch/[a-c]*/**",
+                "torch/d*/**",
+                "torch/[e-n]*/**",
+                "torch/[o-z]*/**",
```

Comment:
> @XuehaiPan Multiple PRs are good for this. I'm saying can you make them normal PRs, not part of the stack? Because, for example, PR number 6 doesn't really depend on PR number 5 - they can be landed independently.

@kit1980 Each PR removes a line here. This causes merge conflict when two PRs modify two consecutive lines of code.